packages/prism-theme-odyssey/src/utils/CustomPropertyInspector.js

```diff
@@ -0,0 +1,122 @@
+const Storage = window.localStorage;
+
+class CustomPropertyInspector {
+  constructor({propertiesElement, inspectorElement}) {
+    this.properties = this.setCustomProperties(propertiesElement);
+
+    this.renderInspector()
+  }
+
+  setCustomProperties(element) {
+    let prop
+    const styles = getComputedStyle(element)
+    const properties = {}
+
+    for (prop in styles) {
+      const name = styles[prop];
+      const value = Storage.getItem(name) || styles.getPropertyValue(name);        
```

Comment:
what if `CustomPropertyInspector` is initialized twice with a different DOM element? Wouldn't the storage item be incorrect here? I'm not sure I understand the local storage caching design here and how it relates to the `propertiesElement`.

Going further, we accept a `propertiesElement` from our constructor to collect custom properties from - but assign overrides to the HTML element. The inheritance of custom properties through the DOM would preclude our updates from working if the propertiesElement was the body for example?

Reply:
bumping this comment just so I'm sure I understand the mechanics here - I haven't yet had a chance to pull this down and play yet 😬